admin-dev/themes/default/scss/admin-theme.scss

```diff
@@ -111,3 +111,7 @@
 
 //Print support
 @import "partials/print";
+
+//modal loader
+@import "../../new-theme/public/new_product_modal";
```

Comment:
I don't think you need this in the new theme At least for now all I see in the `new_product_modal.scss` file are dependencies on bootstrap or UI kit which should already be integrated in the new theme anyway

If you integrated special CSS for the component that would be different but for now there is none, although you'll see in my other comments that it could be enriched but it would still contain some duplicate imports. Even importing product stylesheet directly here can be improved. I would suggest the following structure:

1 - create a `create_product.scss` file containing the common stylesheet useful for both default and new theme (actually this file already exist https://github.com/PrestaShop/PrestaShop/blob/8.0.x/admin-dev/themes/new-theme/scss/pages/product/_create_product_modal.scss#L6-L5)
2 - in `admin-dev/themes/default/scss/partials/_product.scss` import this file, this is how we will update the new theme to handle the modal correctly on any new-theme page (since this CSS is the common one for the whole theme)
3 - create a `create_product_default_theme.scss` file, this one will be used as the entry endpoint for the generated bundle, so it can include the required bootstrap dependencies and of course the `create_product.scss` file as well, this will be used in default theme page so this bundle should be included automatically via the `setMedia` method in default theme pages (as you already did)

Ideally, the default bundle could be "protected" inside a designated scope (see my comment on font.css) to limit the impact, I don't know if it would be possible to also encapsulate the bootstrap dependencies inside a specific ID selector, this way the rules from bootstrap4 wouldn't impact those from bootstrap3 from the default theme

```scss
# That's the idea, maybe this ID selector is too limiting but maybe we can use another container to widen the scope a little, but trying to keep it as precise as possible to limit the side effects
#modal-create-product {
   @import "~bootstrap/scss/mixins/border-radius";
   @import "../../config/settings";
   @import "~prestakit/scss/spinners";
   @import "../../components/modal";
}
```

By following this structure we clearly separate what is inherent to the modal itself and needs to be used in both themes, from the required dependencies that are only needed in the default theme (because the new theme already has them), this way when one day we remove the default theme we can simply remove the `create_product_default_theme.scss` file since it won't be needed anymore, and the new theme will remain unchanged.